.travis.yml

```diff
@@ -1,14 +1,39 @@
-# references:
-# * http://www.objc.io/issue-6/travis-ci.html
-# * https://github.com/supermarin/xcpretty#usage
-
-osx_image: xcode7.3
 language: objective-c
-# cache: cocoapods
-# podfile: Example/Podfile
-# before_install:
-# - gem install cocoapods # Since Travis is not always on latest version
-# - pod install --project-directory=Example
+osx_image: xcode8.1
+env:
+  global:
+    - LC_CTYPE=en_US.UTF-8
+    - PROJECT=SwinjectAutoregistration.xcodeproj
+git:
+  submodules: false
+matrix:
+  include:
+    - env: JOB="LINUX_SPM"
+      os: linux
+      language: generic
+      sudo: required
+      dist: trusty
+      before_install:
+        - eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/9f442512a46d7a2af7b850d65a7e9bd31edfb09b/swiftenv-install.sh)"
+      script:
+        - mv .Package.test.swift Package.swift
+        - swift build
+        - swift test
+    - env: JOB="POD_LINT"
+      script:
+        - pod lib lint
+    - env: JOB="XCODE" DEST="OS=8.4,name=iPhone 5" SCHEME="SwinjectAutoregistration-iOS" SDK="iphonesimulator" ACTION="test"
```

Comment:
Changing the device to iPhone 5s looks ok to fix the error on Travis with Nimble.
https://travis-ci.org/Swinject/SwinjectAutoregistration/jobs/196326139

```
DEST="OS=8.4,name=iPhone 5s" 
```